components/ratings/Ratings.module.css

```diff
@@ -0,0 +1,17 @@
+.stars {
+  display: inline;
```

Comment:
Formating issue
You should reformat the code for the whole file

Reply:
Okay, done.